### PR TITLE
[WX-1842] Remove Genomics Backend

### PR DESCRIPTION
**Pull request by THWiseman**
### Description
Google has shut down their Genomics (a.k.a. PAPI v2Alpha1) API, this PR cleans up associated code. This is not to be confused with the Cloud Life Sciences API (a.k.a. PAPI v2beta, deprecated, but still in use for the next few months) or the Google Batch backend (the new hotness). 

#### `CHANGELOG.md`
 - [x] I updated `CHANGELOG.md` in this PR
 - [ ] I assert that this change shouldn't be included in `CHANGELOG.md` because it doesn't impact community users

#### Terra Release Notes
 - [ ] I added a suggested release notes entry in this Jira ticket
 - [x] I assert that this change doesn't need Jira release notes because it doesn't impact Terra users

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## 88 Release Notes
 
+### Removal of Google Genomics Backend
+Code relating to the Google Genomics API has been removed since Google has entirely disabled that service.
+Cloud Life Sciences (deprecated) and Google Batch (recommended) remain the two viable GCP backends.
```
Comment (aednichols) [inline CHANGELOG.md:7]: We have a `### Removals` heading at the bottom of this release (and we should do a release soon, there's a lot of stuff)

**Review by aednichols — APPROVED** [review]
LGTM with one comment on release notes, a classic battleground of bikeshedding.

File: CHANGELOG.md (hunk 1 above)
Comment (jgainerdewar) [reply CHANGELOG.md:7]: More bikeshedding, I would include the Cromwell names (`v1aplha`, `v2beta`, `batch`) here in addition to the GCP product names, to reduce confusion.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -53,8 +53,6 @@ object Dependencies {
   private val googleCloudNioV = "0.127.18"
   private val googleCloudStorageV = "2.17.2"
   private val googleGaxGrpcV = "2.48.0"
-  // latest date via: https://mvnrepository.com/artifact/com.google.apis/google-api-services-genomics
-  private val googleGenomicsServicesV2Alpha1ApiV = "v2alpha1-rev20210811-1.32.1"
```
Comment (jgainerdewar) [inline project/Dependencies.scala:57]: 👋

**Review by jgainerdewar — APPROVED** [review]
Assuming CI passes, LGTM!

File: centaur/src/main/scala/centaur/test/Test.scala [hunk 3]
```diff
@@ -122,8 +123,8 @@ object Operations extends StrictLogging {
   // The project from the config or from the credentials. By default the project is read from the system environment.
   lazy val projectOption: Option[String] = confProjectOption orElse credentialsProjectOption
 
-  lazy val genomics: Genomics = {
-    val builder = new Genomics.Builder(
+  lazy val genomics: CloudLifeSciences = {
```
Comment (mcovarr) [inline centaur/src/main/scala/centaur/test/Test.scala:126]: Technically this is a `CloudLifeSciences` now and not a `Genomics`, even if neither of those terms really makes sense as an abstraction...

File: services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala [hunk 4]
```diff
@@ -151,14 +150,17 @@ object WorkbenchHealthMonitorServiceActor {
...
+                                    httpInitializer(credentials)
+      )
         .setApplicationName(applicationName)
         .setRootUrl(endpointUrl.toString)
         .build
 
     override def check = Future {
       // https://cloud.google.com/genomics/reference/rest/#rest-resource-v2alpha1projectsoperations
-      genomics.projects().operations().list(s"projects/$papiProjectId/operations").setPageSize(1).execute()
+      genomics.projects().locations().operations().list(s"projects/$papiProjectId/operations").setPageSize(1).execute()
```
Comment (mcovarr) [inline services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala:163]: this is all in a `GenomicsCheckerV2Alpha1` which should go away now

**Review by mcovarr — COMMENTED** [review]
you might want to do a (case insensitive, non word) search for 'v2alpha1'... there's quite a few non-historical references left in docs, comments, and at least one Centaur test

**Comment by aednichols** [comment]
We had a conversation somewhere, possibly Slack DMs, where we discussed multiple PRs that address progressively expanding scopes.

**Pull request merged**